### ISSUE TITLE
Review & prioritize NRRD accessibility violations

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#2511]
The AMP NRRD violations are more complex than the ONRR. We'd like to look at them with Jeff and Maroya to determine the best way for creating and prioritizing issues.

**Comment by eelzi-ONRR** [comment]
Link to the OCIO/AMP report platform: https://departmentofinterior.levelaccess.us/login.php

Exported list of violations: https://doimspp.sharepoint.com/:x:/s/NRRD/Ea-CVbPsGDpDkDLSIe6kZS8BNeTJ7C8_M7yRS86t6VEU2g?e=HSxke5

**Comment by eelzi-ONRR** [comment]
Made 9 issues - one for each violation in the report
